.github/workflows/CI.yml

```diff
@@ -29,63 +29,64 @@ jobs:
-          if [ $security -gt 0 ]
-          then
-            echo "Security plugin is available"
-            ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster" -Dhttps=true -Duser=admin -Dpassword=admin
-          else
-            echo "Security plugin is NOT available"
-            ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster"
-          fi
-      - name: Run AD Backwards Compatibility Tests
-        run: |
-          echo "Running backwards compatibility tests ..."
-          ./gradlew bwcTestSuite -Dtests.security.manager=false
+#        ToDo: Check infra team for 2.0 docker availability and re-enable  
+#       - name: Pull and Run Docker
+#         run: |
+#           plugin=`ls build/distributions/*.zip`
+#           version=2.0.0
+#           plugin_version=2.0.0.0-SNAPSHOT
+#           echo Using OpenSearch $version with AD $plugin_version
+#           cd ..
+#           if docker pull opensearchstaging/opensearch:$version
+#           then
+#             echo "FROM opensearchstaging/opensearch:$version" >> Dockerfile
+#             echo "RUN if [ -d /usr/share/opensearch/plugins/opensearch-anomaly-detection ]; then /usr/share/opensearch/bin/opensearch-plugin remove opensearch-anomaly-detection; fi" >> Dockerfile
+#             echo "ADD anomaly-detection/build/distributions/opensearch-anomaly-detection-$plugin_version.zip /tmp/" >> Dockerfile
+#             echo "RUN /usr/share/opensearch/bin/opensearch-plugin install --batch file:/tmp/opensearch-anomaly-detection-$plugin_version.zip" >> Dockerfile
+#             docker build -t opensearch-ad:test .
+#             echo "imagePresent=true" >> $GITHUB_ENV
+#           else
+#             echo "imagePresent=false" >> $GITHUB_ENV
+#           fi
+#       - name: Run Docker Image
+#         if: env.imagePresent == 'true'
+#         run: |
+#           cd ..
+#           docker run -p 9200:9200 -d -p 9600:9600 -e "discovery.type=single-node" opensearch-ad:test
+#           sleep 90
+#       - name: Run AD Test
+#         if: env.imagePresent == 'true'
+#         run: |
+#           security=`curl -XGET https://localhost:9200/_cat/plugins?v -u admin:admin --insecure |grep opensearch-security|wc -l`
+#           if [ $security -gt 0 ]
+#           then
+#             echo "Security plugin is available"
+#             ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster" -Dhttps=true -Duser=admin -Dpassword=admin
+#           else
+#             echo "Security plugin is NOT available"
+#             ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster"
+#           fi
+#       - name: Run AD Backwards Compatibility Tests
```

Comment:
Should this also be commented for now?

Reply:
There is depenedency on job-scheduler-2.0.0.0..which is currently alpha stage.Will be better to uncomment together hen we have the tested job-scheduler code